firefox-ios/Client/Frontend/Autofill/Address/Edit/EditAddressViewController.swift

```diff
@@ -16,6 +16,8 @@ class EditAddressViewController: UIViewController, WKNavigationDelegate, WKScrip
         return webView
```

Comment:
translatesAutoresizingMaskIntoConstraints can be added in the lazy var here, unless you are seeing issues